packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -28,28 +54,72 @@ export const styles = (theme) => ({
     ...theme.typography.body2,
     minHeight: 'auto',
   },
+  /* Styles applied to a Menu Item's children when a subMenu is present */
+  subMenuItemWrapper: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  /* Styles applied to the subMenuIcon when it is present */
+  subMenuIcon: {
+    marginLeft: theme.spacing(2),
+  },
+  /* Styles applied to subMenuIcon when dirction is 'rtl' */
+  rtlSubMenuIcon: {
+    transform: 'rotate(-180deg)',
+  },
 });
 
 const MenuItem = React.forwardRef(function MenuItem(props, ref) {
+  const theme = useTheme();
+
   const {
+    children,
     classes,
     className,
     component = 'li',
     disableGutters = false,
+    handleArrowRightKeydown,
+    key,
```

Comment:
This will never be available. React special cases the `key` property.

Reply:
Ah, yeah. Nice catch! Not sure how I missed that.